static/app/components/replays/player/replayPlayer.tsx

```diff
@@ -79,6 +79,7 @@ function useReplayerInstance() {
 
 interface Props extends HTMLAttributes<HTMLDivElement> {
   css?: Interpolation<Theme>;
+  ['data-inspectable']?: boolean;
```

Comment:
This was allowed because we extend `HTMLAttributes<HTMLDivElement>` and any `data-` is allowed, but i wanted to be more specific about the type so it's listed, and it's boolean.